easybuild/easyconfigs/m/M4/M4-1.4.17-foss-2015b.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'M4'
+version = '1.4.17'
+
+homepage = 'http://www.gnu.org/software/m4/m4.html'
+description = """GNU M4 is an implementation of the traditional Unix macro processor. It is mostly SVR4 compatible
+  although it has some extensions (for example, handling more than 9 positional parameters to macros).
+ GNU M4 also has built-in functions for including files, running shell commands, doing arithmetic, etc."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
```

Comment:
why not just drop back to GNU 4.9.3-2.25 for this. It will be helpful for other toolchains, like PGI.